packages/cyberstorm/src/components/Toast/ToastContainer.tsx

```diff
@@ -0,0 +1,35 @@
+"use client";
+import { Toast } from "./Toast";
+import styles from "./Toast.module.css";
+import * as RadixToast from "@radix-ui/react-toast";
+
+export function ToastContainer(props: {
+  toasts: {
+    id: string;
+    variant?: "info" | "danger" | "warning" | "success";
+    icon?: JSX.Element;
+    message?: string;
+    timer?: number;
+  }[];
+}) {
+  const { toasts } = props;
+
+  return (
+    <RadixToast.Viewport asChild>
+      <div className={styles.toastContainer}>
+        {toasts.map((toast) => (
+          <Toast
```

Comment:
If the prop interfaces would match, this could be reduced to `<Toast key={toast.id} {...toast} />`